app/views/users/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<!--<form action="/users" method="post" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username</label>
+  <input type="text" name="user[username]" id="username">
+  <label for="email">Email</label>
+  <input type="email" name="user[email]" id="email">
+  <label for="password">Password</label>
+  <input type="password" name="user[password]" id="password">
+  <input type="submit" value="Submit">
+</form> -->
+
+<!--<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email") %>
+  <%= email_field_tag(:email) %>
+  <%= label_tag(:password, "Password") %>
+  <%= password_field_tag(:password) %>
```

Comment:
- [OPTIONAL] At the moment the form fails to submit. instead of a symbol: username, try to pass in user[username] just like the previous form.